docs/operations/clean-metadata-store.md

```diff
@@ -23,20 +23,32 @@ description: "Defines a strategy to maintain Druid metadata store performance by
   ~ specific language governing permissions and limitations
   ~ under the License.
   -->
-When you delete some entities from Apache Druid, records related to the entity may remain in the metadata store including:
 
-- segments records
-- audit records
-- supervisor records
-- rule records
-- compaction configuration records
-- datasource records created by supervisors
+Apache Druid relies on [metadata storage](../dependencies/metadata-storage.md) to track information on data storage, operations, and system configuration.
+The metadata store includes the following:
 
-If you have a high datasource churn rate, meaning you frequently create and delete many short-lived datasources or other related entities like compaction configuration or rules, the leftover records can start to fill your metadata store and cause performance issues.
+- Segment records
+- Audit records
+- Supervisor records
+- Rule records
+- Compaction configuration records
+- Datasource records created by supervisors
+- Indexer task logs
+
+When you delete some entities from Apache Druid, records related to the entity may remain in the metadata store.
+If you have a high datasource churn rate, meaning you frequently create and delete many short-lived datasources or other related entities like compaction configuration or rules, the leftover records can fill your metadata store and cause performance issues.
+To maintain metadata store performance, you can configure Apache Druid to automatically remove records associated with deleted entities from the metadata store.
+
+By default, Druid automatically cleans up metadata older than 90 days.
+This applies to all metadata entities in this topic except compaction configuration records, which do not have a retention period, and indexer task logs, for which cleanup is disabled by default.
```

Comment:
Might be worth pointing out the behavior for compaction configuration record here since the durationToRetain does not applies.

Reply:
The behavior is that at every cycle of the metadata management tasks, any compaction configuration of datasources that is inactive (datasource with no used and unused segments) will be deleted.

Reply:
```suggestion
This applies to all metadata entities in this topic except compaction configuration records, which do not have a retention period, and indexer task logs, for which cleanup is disabled by default.
Druid deletes compaction configuration records at every cleanup cycle for inactive datasources.

```